src/main/java/org/jvnet/hudson/test/BuildWatcher.java

```diff
@@ -24,150 +24,33 @@
 
 package org.jvnet.hudson.test;
 
-import hudson.Extension;
-import hudson.console.LineTransformationOutputStream;
-import hudson.model.Run;
-import hudson.model.TaskListener;
-import hudson.model.listeners.RunListener;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.io.PrintStream;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
-import jenkins.model.Jenkins;
 import org.junit.rules.ExternalResource;
+import org.jvnet.hudson.test.fixtures.BuildWatcherFixture;
 
 /**
- * Echoes build output to standard error as it arrives.
- * Usage: <pre>{@code @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();}</pre>
- * Should work in combination with {@link JenkinsRule} or {@link RestartableJenkinsRule}.
- * @see JenkinsRule#waitForCompletion
- * @see JenkinsRule#waitForMessage
- * @see TailLog
+ * This is the JUnit 4 implementation of {@link BuildWatcherFixture}.
+ * Usage: <pre>{@code
+ * @ClassRule
+ * public static final BuildWatcherExtension BUILD_WATCHER = new BuildWatcher();
```

Comment:
conventionally written as
```suggestion
 * public static final BuildWatcherExtension buildWatcher = new BuildWatcher();
```
since it is not a constant

Reply:
But it is an constant? Am I missing something?